src/lwc/geTemplateBuilderFormFields/geTemplateBuilderFormFields.js

```diff
@@ -58,15 +71,27 @@ export default class geTemplateBuilderFormFields extends LightningElement {
         let missingRequiredFieldMappings = [];
 
         const elements = this.template.querySelectorAll('lightning-input');
+        let counter = 0;
+
         elements.forEach(el => {
-            if (el.required && !el.checked) {
+            if (REQUIRED_FORM_FIELDS.includes(el.getAttribute('data-source-api-name')) && el.checked) {
+              counter++;
+            }
+            if (el.required && !el.checked ||
+                el.getAttribute('data-source-api-name') === DONATION_DONOR_INFO.fieldApiName && !el.checked) {
```

Comment:
Could this be simplified a bit as "!el.checked && (el.required  || el.getAttribute('data-source-api-name') === DONATION_DONOR_INFO.fieldApiName)"?

Reply:
Absolutely!